app/views/allocations/confirm.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= form_tag(allocations_path, method: :post, id: "confirm_allocation_form") do %>
       <h1 class="govuk-heading-xl govuk-!-margin-top-4">Confirm allocation</h1>
       <p class="govuk-body">You are allocating <%= "#{@prisoner.first_name} #{@prisoner.last_name}".titleize %> to <%= "#{@pom.first_name} #{@pom.last_name}".titleize %>.</p>
-      <p class="govuk-body">We will send a confirmation email to POM-EMAIL-ADDRESS.</p>
+      <p class="govuk-body">We will send a confirmation email to <%= @pom.emails.first %>.</p>
```

Comment:
What does it do if `@poms.emails.empty?` ?

Reply:
At the moment it displays "We will send a confirmation email to .", which isn't ideal.  However, I've created a ticket asking Leigh to provide some content, and spoken to Jay about the idea of trying to catch POMs not having email addresses earlier in the process.  I'm happy to put something in temporarily?  I can also add a `return if @poms.emails.empty` line into the PomMailer to stop it trying to send messages?

Reply:
Yeah sounds good.  Just add something temporary (or even just remove that line entirely) and add it to the ticket for Leigh?  Having the mailer be a bit more defensive when it won't be able to send mail could be a good thing, but also the error telling us a user doesn't have an email is useful.  I'd say for now, go with being defensive.